aloha-io-avro/src/test/scala/com/eharmony/aloha/audit/impl/avro/ImplicitsTest.scala

```diff
@@ -0,0 +1,79 @@
+package com.eharmony.aloha.audit.impl.avro
+
+import com.google.common.collect.Lists
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.BlockJUnit4ClassRunner
+import scala.collection.JavaConverters.seqAsJavaListConverter
+import com.eharmony.aloha.audit.impl.avro.Implicits.{RichFlatScoreList, RichScore}
+
+import java.{lang => jl, util => ju}
+
+/**
+  * Created by ryan.deak on 7/5/17.
+  */
+@RunWith(classOf[BlockJUnit4ClassRunner])
+class ImplicitsTest {
+  import ImplicitsTest._
+  import FlatScoreListTest.flatScoreList
+
+  @Test def testFlatScoreListToScore(): Unit =
+    assertEquals(score, flatScoreList.toScore)
+
+  @Test def testScoreToFlatScoreList(): Unit =
+    assertEquals(flatScoreList, score.toFlatScoreList)
+
+  /**
+    * Tests that scores are corrected inserted during conversions and that
+    * children ordering is preserved.
+    */
+  @Test def testRoundTripConversionFromScore(): Unit =
+    assertEquals(score, score.toFlatScoreList.toScore)
+
+  @Test def testRoundTripConversionFromFlatScoreList(): Unit =
+    assertEquals(flatScoreList, flatScoreList.toScore.toFlatScoreList)
+
+  /**
+    * Unlike `testRoundTripConversionFromScore`, this tests verifies that
+    * all fields in scores are correctly copied during conversions.
+    */
+  @Test def testAllFieldsAppear(): Unit = {
```

Comment:
Added this test to ensure all fields are copied.  In the process found and fixed a bug in the conversion code.